tests/unit/lms/views/api/lti_test.py

```diff
@@ -7,36 +7,36 @@
 
 from lms.views.api.lti import CanvasPreRecordHook, LTIOutcomesViews
 
-pytestmark = pytest.mark.usefixtures("lti_outcomes_client")
+pytestmark = pytest.mark.usefixtures("lti_grading_service")
 
 
 class TestRecordCanvasSpeedgraderSubmission:
     GRADING_ID = "lis_result_sourcedid"
```

Comment:
I think a sentinel would be better here again. The string `lis_result_sourcedid` is a bit misleading